client/app/components/project/package-list-item.hbs

```diff
@@ -110,8 +110,9 @@
     (if (eq @package.dcpPackagetype (optionset 'package' 'dcpPackagetype' 'code' 'DRAFT_EAS')) 'draft-eas.edit' 'draft-eas.edit') --}}
```

Comment:
realized my comment here was wrong, it should have said ...'final-eas.edit' at the end 😅  
Thanks taylor! 
Can you also delete these (poorly written) comments?

Reply:
Sure :)